material-overrides/partials/content.html

```diff
@@ -1,12 +1,22 @@
 {#-
-    This file was automatically generated - do not edit
-  -#}
-  {% if "material/tags" in config.plugins and tags %}
-    {% include "partials/tags.html" %}
-  {% endif %}
-  {% if "\x3ch1" not in page.content %}
-  <h1>{{ page.title | d(config.site_name, true)}}</h1>
-  {% endif %}
-  {{ page.content }}
-  {% include "partials/source-file.html" %}
-  {% include "partials/comments.html" %}
\ No newline at end of file
+  This file was automatically generated - do not edit
+-#}
+
+{% if "material/tags" in config.plugins and tags %}
+  {% include "partials/tags.html" %}
+{% endif %}
+
+{% set badge_html = '' %}
+{% if page.meta and page.meta.tutorial_badge %}
+  {% set badge_html = '<span class="badge ' ~ (page.meta.tutorial_badge | lower | replace(".", "-")) ~ '">' ~ (page.meta.tutorial_badge | capitalize) ~ '</span>' %}
+  {{ badge_html | safe }}
```

Comment:
I think we should try to see what our other options are because I'd really prefer to see the badge under the title. We can probably look at modifying the `page.content` a little bit